### PR TITLE
Use arraycmplen opcode

### PR DESCRIPTION
**Pull request by Spencer-Comin**
`arraycmp` opcode with `arrayCmpLen` flag set is replaced with the opcode `arraycmplen` in https://github.com/eclipse/omr/pull/6983.

### CODE REVIEW

**Comment by Spencer-Comin** [comment]
This has passed internal jenkins sanity tests on Z, x86, Power, and AArch64

File: runtime/compiler/optimizer/IdiomTransformations.cpp [hunk 1]
```diff
@@ -9351,6 +9351,14 @@ CISCTransform2ArrayCmp(TR_CISCTransformer *trans)
          generateArraycmplen = true;
          }
       }
+   if (generateArraycmplen)
+      {
+      if (!comp->cg()->getSupportsArrayCmpLen())
+         {
+         // arrayCmpLen is not supported by codegen
+         generateArraycmplen = false;
```
Comment (jdmpapin) [inline runtime/compiler/optimizer/IdiomTransformations.cpp:9359]: It's not clear to me that we can simply elect not to use `arraycmplen` despite the determination of the above logic. Instead of setting `generateArraycmplen` to false, I think we should consider the length result to be potentially required, and so refuse to transform in this situation

Comment (jdmpapin) [reply runtime/compiler/optimizer/IdiomTransformations.cpp]: Thanks for updating this! It should work, though I did like the previous code structure that tested for support in only one place. Now the `getSupportsArrayCmpLen()` condition is repeated in 4 different places, which to me doesn't read as clearly. I also think it would be good to trace when giving up regardless of why `generateArraycmplen` was set, and the single test you had before would have been a good place to do that. What do you think?

**Comment by jdmpapin** [comment]
Please see my follow-up response to my earlier review comment - mentioning it here just for visibility

**Comment by jdmpapin** [comment]
Is this in draft state just in order to wait for https://github.com/eclipse/omr/pull/6983 to be merged first? In case it is, the two PRs will actually need a coordinated merge

**Review by jdmpapin — APPROVED** [review]

**Comment by jdmpapin** [comment]
Jenkins test sanity.functional,sanity.openjdk all jdk8,jdk11,jdk17,jdk20 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
Please look into the 32-bit Windows failures, which look related to these changes. They're crashing while compiling methods that can be expected to involve `arraycmplen`:
- `jit/test/loopReduction/longMemCmp.test`
- `com/ibm/jit/JITHelpers.intrinsicIndexOfStringUTF16`

Note that `Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk` is not actually related to these crashes. It shows up in the stack trace because of a lack of symbols in the code that's actually running.

Probably codegen will need to be adjusted for x86-32.

----

Other failures so far:

The Mac failure is #11930.

The AIX builds failed with "Deadlock condition if locked," which seems to be #8625.

The z Linux failure is a timeout in a CRIU test (cmdLineTester_criu_nonPortableRestore_Xdump_events_4):
```
Testing: Restore trace options test with -Xtrace before checkpoint - 2
...
***[TEST INFO 2023/06/20 01:06:47] ProcessKiller detected a timeout after 300000 milliseconds!***
...
***[TEST INFO 2023/06/20 01:07:47] executing kill -ABRT 1242972***
***[TEST INFO 2023/06/20 01:07:47] kill -ABRT signal sent***
Time spent executing: 360334 milliseconds
Test result: FAILED
```
I don't see that this particular issue is known, but I can scarcely imagine that it's related to these changes.

**Comment by Spencer-Comin** [comment]
Reverting this PR (and associated OMR PR https://github.com/eclipse/omr/pull/6983) to draft while I resolve the failing tests

**Comment by Spencer-Comin** [comment]
I did some tweaking to this PR and https://github.com/eclipse/omr/pull/6983 to resolve failures in x86-32, it should be good to go now

File: runtime/compiler/optimizer/IdiomTransformations.cpp [hunk 2]
```diff
@@ -8979,8 +8979,8 @@ makeMixedMemSetGraph(TR::Compilation *c, int32_t ctrl)
 //        ImportantNode(5) - increment the array index for src2
 //        ImportantNode(6) - the size of elements (NULL for byte arrays)
 //
-// Note: If we need to know the position where characters are different (flag generateArraycmplen),
-//       we generate the CLCL instruction. Otherwise, we generate the CLC instruction.
+// Note: If we need to know the position where characters are different, we generate arraycmplen.
+//       Otherwise, we generate arraycmp.
```
[the remark this answers is not in the document]
Comment (Spencer-Comin) [reply runtime/compiler/optimizer/IdiomTransformations.cpp]: I just removed the note

**Review by jdmpapin — APPROVED** [review]

**Comment by jdmpapin** [comment]
Jenkins test sanity.functional,sanity.openjdk all jdk8,jdk11,jdk17,jdk21 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
[Test_openjdk11_j9_sanity.openjdk_s390x_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.openjdk_s390x_linux_Personal/46/) hit "Unexpected exit from test [exit code: 137]" in java/lang/StringBuilder/HugeCapacity (#17323)

[Test_openjdk21_j9_sanity.openjdk_aarch64_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk21_j9_sanity.openjdk_aarch64_linux_Personal/6/) hit a timeout in Short128VectorTests (#17937)

[Test_openjdk21_j9_sanity.openjdk_s390x_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk21_j9_sanity.openjdk_s390x_linux_Personal/5/) hit a crash in java/util/concurrent/locks/Lock/OOMEInAQS (#16659)

**Comment by Spencer-Comin** [comment]
[Test_openjdk11_j9_sanity.openjdk_ppc64le_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.openjdk_ppc64le_linux_Personal/21/) segfaults in jdk/internal/loader/ClassloaderValue/ClassloaderValueTest (#12556)

[Test_openjdk11_j9_sanity.openjdk_x86-64_windows_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.openjdk_x86-64_windows_Personal/26/) fails with "PortUnreachableException not thrown" in sun/security/krb5/auto/Unreachable (#17743)

[Test_openjdk17_j9_sanity.functional_x86-64_windows_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk17_j9_sanity.functional_x86-64_windows_Personal/290/) appears to be hitting an [infrastructure issue](https://openj9-jenkins.osuosl.org/job/Test_openjdk17_j9_sanity.functional_x86-64_windows_Personal_testList_1/283/console#:~:text=Cannot%20contact%20win2019x64%2Dopenj9%2D1%3A%20hudson.remoting.ChannelClosedException%3A%20Channel%20%22hudson.remoting.Channel%40983b4e7f%3Awin2019x64%2Dopenj9%2D1%22%3A%20Remote%20call%20on%20win2019x64%2Dopenj9%2D1%20failed.%20The%20channel%20is%20closing%20down%20or%20has%20closed%20down)

[Test_openjdk21_j9_sanity.openjdk_ppc64le_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk21_j9_sanity.openjdk_ppc64le_linux_Personal/3/) hit timeouts in Byte64VectorTests and Short128VectorTests (#17937), and also hit a timeout in java/lang/Class/forName/modules/TestDriver that I will investigate further

[Test_openjdk21_j9_sanity.openjdk_x86-64_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk21_j9_sanity.openjdk_x86-64_windows_Personal/5/) hit timeouts in Short128VectorTests, Byte64VectorTests, and Int128VectorTests (#17937)

[Test_openjdk21_j9_sanity.openjdk_x86-64_windows_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk21_j9_sanity.openjdk_x86-64_windows_Personal/5/) hit timeouts in java/security/SecureRandom/ApiTest and java/security/SecureRandom/EnoughSeedTest (#16734)

**Comment by Spencer-Comin** [comment]
I have been unable to reproduce the timeout in https://openj9-jenkins.osuosl.org/job/Test_openjdk21_j9_sanity.openjdk_ppc64le_linux_Personal/3/.

@jdmpapin given that a timeout seems an unlikely failure mode for `arraycmplen`, and that the java/lang/Class/forName/modules/TestDriver test source doesn't appear to have any unusual looking for-loops that could possibly trigger some `arraycmplen` corner case, I'm assuming this failure is unrelated to my changes.

**Comment by jdmpapin** [comment]
Jenkins test sanity.functional,sanity.openjdk xmac jdk17 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
Jenkins test sanity.functional,sanity.openjdk aix jdk21 depends eclipse/omr#6983

**Comment by Spencer-Comin** [comment]
[Test_openjdk11_j9_sanity.openjdk_s390x_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.openjdk_s390x_linux_Personal/47/) had a failure in `jdk_lang_j9_0` that was seen in the previous build from https://github.com/eclipse-openj9/openj9/issues/17323

All the other failures are `"TEST RESULT: Failed. Execution failed: `main' threw exception: java.lang.RuntimeException: java.util.zip.ZipException: Invalid CEN header (invalid zip64 extra data field size)"`

**Comment by jdmpapin** [comment]
The failure in `ReadNonStandardExtraHeadersTest` seems like it could be related. It failed quite a few times, it doesn't look to be a known issue, and at a very abstract level compression could be related to `arraycmplen`. If it is related, though, then it's weird that it didn't show up before. Could you please take a look into that?

**Comment by Spencer-Comin** [comment]
The ```"TEST RESULT: Failed. Execution failed: `main' threw exception: java.lang.RuntimeException: java.util.zip.ZipException: Invalid CEN header (invalid zip64 extra data field size)"``` error is 100% reproducible without my changes. Interestingly, the error goes away when moving from jdk-17.0.9+3 to jdk-17.0.9+4.

## Internal builds
- [jdk-17.0.9+3](https://hyc-runtimes-jenkins.swg-devops.com/job/Pipeline-Build-Test-Personal/17891/)
- [jdk-17.0.9+4](https://hyc-runtimes-jenkins.swg-devops.com/job/Pipeline-Build-Test-Personal/17892/)

## Internal Grinders
### jdk-17.0.9+3
- [zLinux](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/34506/)
- [xLinux](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/34507/)
- [macOS](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/34510/)

### jdk-17.0.9+4
- [zLinux](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/34509/)
- [xLinux](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/34508/)
- [macOS](https://hyc-runtimes-jenkins.swg-devops.com/job/Grinder/34505/)

**Comment by jdmpapin** [comment]
OK, thanks. Please resolve the conflict

**Comment by jdmpapin** [comment]
The conflict resolution has undone the replacement of `getSupportsLoweringConst[IL]Div()` with `getSupports[IL]MulHigh()` (from #17892) in `TR_CISCGraph::initializeGraphs()`. Could you please restore the new names?

**Comment by jdmpapin** [comment]
Here we go again

Jenkins test sanity.functional,sanity.openjdk all jdk8,jdk11,jdk17,jdk21 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
ddrgen failed on Windows. This has been observed recently in the OMR acceptance build - see #18015

Jenkins test sanity.functional,sanity.openjdk win jdk21 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
The [z Linux JDK11 sanity.openjdk failure](https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.openjdk_s390x_linux_Personal/48/) is in java/nio/Buffer/Chew.java
```
[2023-08-28T19:20:25.270Z] java.lang.OutOfMemoryError: Direct buffer memory
[2023-08-28T19:20:25.270Z] 	at java.base/java.nio.Bits.reserveMemory(Bits.java:175)
[2023-08-28T19:20:25.270Z] 	at java.base/java.nio.DirectByteBuffer.<init>(DirectByteBuffer.java:123)
[2023-08-28T19:20:25.270Z] 	at java.base/java.nio.ByteBuffer.allocateDirect(ByteBuffer.java:317)
```
This is #17723

Jenkins test sanity.openjdk zlinux jdk11 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
The [x86 Mac JDK17 sanity.functional failure](https://openj9-jenkins.osuosl.org/job/Test_openjdk17_j9_sanity.functional_x86-64_mac_Personal/273/) is SharedCPEntryInvokerTests_2 crashing here:
```
getJNIMethodID+0x6c (0x0000000004F841BC [libj9vm29.dylib+0x401bc])
prepareToFixMemberNamesObjectIteratorCallback+0x158 (0x0000000005C186B8 [libj9jvmti29.dylib+0x336b8])
j9mm_iterate_region_objects+0xdd (0x000000000552E60D [libj9gc29.dylib+0xe560d])
j9mm_iterate_all_objects+0xbb (0x000000000552F40B [libj9gc29.dylib+0xe640b])
prepareToFixMemberNames+0x46 (0x0000000005C18546 [libj9jvmti29.dylib+0x33546])
redefineClassesCommon+0x31b (0x0000000005BECFCB [libj9jvmti29.dylib+0x7fcb])
jvmtiRedefineClasses+0xc2 (0x0000000005BECC42 [libj9jvmti29.dylib+0x7c42])
Java_com_ibm_jvmti_tests_redefineClasses_rc001_redefineClass+0xb1 (0x0000000005D4E941 [libjvmtitest.dylib+0x13941])
```
I'll look into this one myself

**Comment by jdmpapin** [comment]
Jenkins test sanity.functional,sanity.openjdk all jdk8,jdk11,jdk17,jdk21 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
[Test_openjdk21_j9_sanity.openjdk_aarch64_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk21_j9_sanity.openjdk_aarch64_linux_Personal/10/) failed in `java/foreign/loaderLookup/TestLoaderLookup.java`:
```
[2023-09-05T17:27:12.594Z] java.lang.reflect.InvocationTargetException
[2023-09-05T17:27:12.594Z] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:115)
[2023-09-05T17:27:12.594Z] 	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
[2023-09-05T17:27:12.594Z] 	at TestLoaderLookup.main(TestLoaderLookup.java:57)
[2023-09-05T17:27:12.594Z] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
[2023-09-05T17:27:12.594Z] 	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
[2023-09-05T17:27:12.594Z] 	at com.sun.javatest.regtest.agent.MainWrapper$MainTask.run(MainWrapper.java:138)
[2023-09-05T17:27:12.594Z] 	at java.base/java.lang.Thread.run(Thread.java:1595)
[2023-09-05T17:27:12.594Z] Caused by: java.lang.NullPointerException: Cannot invoke "java.util.Optional.empty()" because "mapper" is null
[2023-09-05T17:27:12.594Z] 	at java.base/java.util.Optional.map(Optional.java:258)
[2023-09-05T17:27:12.594Z] 	at java.base/jdk.internal.foreign.abi.AbstractLinker.stripNames(AbstractLinker.java:279)
[2023-09-05T17:27:12.594Z] 	at java.base/jdk.internal.foreign.abi.AbstractLinker.downcallHandle0(AbstractLinker.java:102)
[2023-09-05T17:27:12.594Z] 	at java.base/jdk.internal.foreign.abi.AbstractLinker.downcallHandle(AbstractLinker.java:88)
[2023-09-05T17:27:12.594Z] 	at invoker.Invoker.invoke(Invoker.java:31)
[2023-09-05T17:27:12.594Z] 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
[2023-09-05T17:27:12.594Z] 	... 6 more
```
This is #18074. The issue says it's a segmentation fault, but at least [this failure](https://openj9-jenkins.osuosl.org/job/Test_openjdk21_j9_sanity.openjdk_aarch64_linux_Nightly_testList_1/46/consoleText) linked from the issue manifests as an `InvocationTargetException` caused by NPE

[Test_openjdk21_j9_sanity.openjdk_s390x_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk21_j9_sanity.openjdk_s390x_linux_Personal/8/) crashed in `OOMEInAQS` (#16659)

[Test_openjdk21_j9_sanity.openjdk_x86-64_windows_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk21_j9_sanity.openjdk_x86-64_windows_Personal/8/) timed out in `java/lang/Thread/virtual/stress/Skynet.java#default` (#16728)

Jenkins test sanity.openjdk all jdk21 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
Jenkins test sanity.openjdk alinux,zlinux,win jdk21 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
Jenkins test sanity.openjdk alinux64,zlinux,win jdk21 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
[Test_openjdk11_j9_sanity.functional_x86-64_windows_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.functional_x86-64_windows_Personal/290/) failed with an off-by-1 value in `testThreadMXBeanProxy`:
```
[2023-09-05T20:17:45.313Z] FAILED: testThreadMXBeanProxy
[2023-09-05T20:17:45.313Z] java.lang.AssertionError: expected:<25> but was:<24>
```
Looks like #17866

[Test_openjdk11_j9_sanity.functional_x86-64_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.functional_x86-64_linux_Personal/317/) didn't launch properly, and there are no testList_0/1 links in the log, but there is this:
```
Cannot contact ub16x64j91: java.lang.InterruptedException
```
Jenkins test sanity.functional xlinux,win jdk11 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
[Test_openjdk11_j9_sanity.openjdk_x86-64_windows_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.openjdk_x86-64_windows_Personal/29/) failed with `RuntimeException: PortUnreachableException not thrown` in `sun/security/krb5/auto/Unreachable.java`, which is #17743

Jenkins test sanity.openjdk win jdk11 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
[Test_openjdk21_j9_sanity.openjdk_s390x_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk21_j9_sanity.openjdk_s390x_linux_Personal/9/) segfaulted in `OOMEInAQS` again (#16659)

[Test_openjdk21_j9_sanity.openjdk_ppc64_aix_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk21_j9_sanity.openjdk_ppc64_aix_Personal/6/) crashed and timed out in `Skynet.java#default` (#16728), which should now be excluded

Jenkins test sanity.openjdk zlinux,aix jdk21 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
[Test_openjdk11_j9_sanity.openjdk_x86-64_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.openjdk_x86-64_linux_Personal/28/): `java/security/SecureRandom/NoSync.java` timed out while Thread-34 is reseeding something and other threads are waiting for it (#17576)

Jenkins test sanity.openjdk xlinux jdk11 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
[Test_openjdk21_j9_sanity.openjdk_x86-64_mac_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk21_j9_sanity.openjdk_x86-64_mac_Personal/6/) failed to run testList_0 due to an infra problem:
```
Cannot contact mac12-x86-1: java.lang.InterruptedException
```
Jenkins test sanity.openjdk xmac jdk21 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
[Test_openjdk17_j9_sanity.openjdk_aarch64_mac_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk17_j9_sanity.openjdk_aarch64_mac_Personal/18/): testList_1 failed with `Cannot contact mac13-aarch64-8: java.lang.InterruptedException`, but it looks like all tests passed

**Comment by jdmpapin** [comment]
[Test_openjdk8_j9_sanity.openjdk_x86-64_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk8_j9_sanity.openjdk_x86-64_linux_Personal/21/): testList_1 failed with what looks to be an infra problem:
```
19:23:29  Exception: java.io.IOException: Failed to copy file:/var/lib/jenkins/jobs/test.getDependency/builds/846/archive/jython-standalone.jar to /home/jenkins/workspace/Test_openjdk8_j9_sanity.openjdk_x86-64_linux_Personal_testList_1/aqa-tests/TKG/lib/jython-standalone.jar
...
19:23:29  Cannot run copyArtifacts from test.getDependency. Skipping copyArtifacts...
[Pipeline] fileExists
...
19:23:29  Exception: org.jenkinsci.plugins.workflow.support.steps.AgentOfflineException: Unable to create live FilePath for cent6-x64-5; cent6-x64-5 was marked offline: Timed out for last 5 attempts
```
Jenkins test sanity.openjdk xlinux jdk8 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
[Test_openjdk11_j9_sanity.functional_x86-64_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.functional_x86-64_linux_Personal/318/): Cannot contact ub16x64j91

Jenkins test sanity.functional xlinux jdk11 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
[Test_openjdk17_j9_sanity.openjdk_x86-64_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk17_j9_sanity.openjdk_x86-64_linux_Personal/26/): testList_2:
```
AgentOfflineException: Unable to create live FilePath for cent7-x64-1; cent7-x64-1 was marked offline: Timed out for last 5 attempts
```
Jenkins test sanity.openjdk xlinux jdk17 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
[Test_openjdk21_j9_sanity.openjdk_ppc64le_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk21_j9_sanity.openjdk_ppc64le_linux_Personal/6/): `Skynet.java#default` failed

[Test_openjdk21_j9_sanity.openjdk_x86-64_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk21_j9_sanity.openjdk_x86-64_linux_Personal/6/): testList_2:
```
AgentOfflineException: Unable to create live FilePath for ub16x64j91; ub16x64j91 was marked offline: Timed out for last 5 attempts
```
Jenkins test sanity.openjdk xlinux,plinux jdk21 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
These two failed with `Cannot contact cent7-x64-$n: java.lang.InterruptedException` in the build step (Build_JDK{8,17}_x86-64_linux_Personal)

Jenkins test sanity.openjdk xlinux jdk8,jdk17 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
[Test_openjdk17_j9_sanity.functional_x86-64_mac_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk17_j9_sanity.functional_x86-64_mac_Personal/275/): testList_0: `Cannot contact mac10-x86-1: java.lang.InterruptedException`

Jenkins test sanity.functional xmac jdk17 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
[Test_openjdk8_j9_sanity.openjdk_x86-64_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk8_j9_sanity.openjdk_x86-64_linux_Personal/22/): timed out in `java/util/concurrent/BlockingQueue/MultipleProducersSingleConsumerLoops.java` (#10158)

The following jobs were aborted for some reason. Also, oddly, they still show as pending in the list of checks for this PR ("3 pending," "Pending — Build Started"). I might have looked at them sooner if there had been a red X
- [Test_openjdk11_j9_sanity.functional_x86-64_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.functional_x86-64_linux_Personal/319/)
- [Test_openjdk11_j9_sanity.openjdk_x86-64_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.openjdk_x86-64_linux_Personal/29/)
- [Test_openjdk21_j9_sanity.openjdk_x86-64_mac_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk21_j9_sanity.openjdk_x86-64_mac_Personal/7/)

**Comment by jdmpapin** [comment]
Jenkins test sanity.functional,sanity.openjdk xlinux jdk11 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
Jenkins test sanity.openjdk xlinux jdk8 depends eclipse/omr#6983

**Comment by jdmpapin** [comment]
Jenkins test sanity.openjdk xmac jdk21 depends eclipse/omr#6983

**Comment by Spencer-Comin** [comment]
[Test_openjdk11_j9_sanity.functional_x86-64_linux_Personal](https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.functional_x86-64_linux_Personal/320/) failure is unrelated to `arraycmplen`. It's a CRIU failure, failing with ["can't bind server address: Address already in use"](https://openj9-jenkins.osuosl.org/job/Test_openjdk11_j9_sanity.functional_x86-64_linux_Personal/320/tapResults/#:~:text=can%27t%20bind%20server%20address%3A%20Address%20already%20in%20use)
